pages/interface/components/Content/index.js

```diff
@@ -187,11 +199,27 @@ function ViewMode({ setComponentMode, contentObject, viewFrame }) {
               whiteSpace: 'nowrap',
               gap: 1,
               mt: '2px',
+              mr: isOptionsMenuVisible ? '28px' : 0,
               color: 'fg.muted',
             }}>
             <BranchName as="address" sx={{ fontStyle: 'normal' }}>
               <Link href={`/${contentObject.owner_username}`}>{contentObject.owner_username}</Link>
             </BranchName>
+            {isPageRootOwner && (
+              <LabelGroup>
+                <Box sx={{ display: 'flex' }}>
+                  <Label aria-hidden sx={{ opacity: 0, userSelect: 'none' }}>
+                    Autor
+                  </Label>
```

Comment:
Por causa do `overflow: hidden` em um elemento antecessor e da `Tooltip` não ser exibida num Portal, precisei usar uma ~gambiarra~ técnica avançada para exibir dois elementos com `position: absolute` lado a lado (`Label` e `PastTime`). Para usar essa técnica no `PastTime` também, precisaria de um pouco mais de trabalho (deixar o componente renderizar com ou sem `Tooltip`), então eu só diminuí o `margin-right` dele, já que adicionei um `margin-right` no pai.

O `opacity: 0` faz com que eu consiga criar um elemento que tome espaço na tela sem ser visível, e esse espaço é visualmente representado pelo elemento com `position: absolute` (o `Label` com `Tooltip`).

Eu realmente tentei várias alternativas, mas nenhuma dava certo. Se alguém ter uma solução melhor, pode falar.

Parece que tem um [Tooltip v2](https://primer.style/react/drafts/Tooltip) em Draft que o Primer já está usando internamente em alguns componentes e (suspeito que) resolve esse problema.

Do jeito que está estilizado agora, para adicionar uma nova `Label` ao lado de outra, cada uma precisa de um `Box` com `display: flex`.

Reply:
Eu criei o Tooltip customizado de uma forma que não precisa mexer no restante do código, pois assim também fica mais fácil de retirá-lo quando não for mais necessário.

Vou commitar pra você ver o que eu fiz, pois acho que resolveu tudo que comentamos e mais alguns problemas do alinhamento dos textos, mas pode sobrescrever o commit se achar melhor.